### PR TITLE
Created Greek Translation of README.md

### PR DESCRIPTION
**Pull request by MIP2000**
### Description

- Docs

### CODE REVIEW

**Comment by MIP2000** [comment]
I created a translation of the README.md file in Greek language. I am a native speaker. Hope that everything is ok, as i encountered a problem in updating the original README.md in order to add my translation link next to the Japanese one.  Please check it. Thanks a lot.

File: translations/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) |
+| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md)                   ￨ [:el: Greek](README.greek.md)  |
```
Comment (marionschleifer) [inline translations/README.md]: The flag doesn't show: 

image

If you don't find a flag that works, you can remove it.

Comment (MIP2000) [reply translations/README.md]: I added a working flag.

**Comment by marionschleifer** [comment]
@MIP2000 this all looks great 🙌 the only thing left is to add a link to this to the [English version under translations](https://github.com/hasura/graphql-engine/blob/master/README.md#translations). Then we'll be ready to merge 🙂

**Comment by MIP2000** [comment]
Done!!! I hope that it looks okay. If i find some time in the next days, maybe i will translate the next docs.

**Comment by MIP2000** [comment]
Hello, may i submit this PR for Hacktoberfest?

**Comment by marionschleifer** [comment]
@MIP2000 could you please send an email to marion@hasura.io, so that we can arrange the shipping of your swag? 🙂

**Comment by MIP2000** [comment]
Yes, of course. Thanks a lot!

**Pull request merged**